src/main/java/org/humancellatlas/ingest/export/destination/ExportDestination.java

```diff
@@ -0,0 +1,11 @@
+package org.humancellatlas.ingest.export.destination;
+
+import lombok.Data;
+
+@Data
+public class ExportDestination {
+    private final ExportDestinationName name;
+    private final String version;
+    private final Object context;
```

Comment:
Is the reason this is okay on [MetadataDocument](https://github.com/ebi-ait/ingest-core/blob/e90bafc9801c8dcd56c8eb4510655dbcb279384d/src/main/java/org/humancellatlas/ingest/core/MetadataDocument.java#L32) because of the JSON validator?

Reply:
I don't think it's okay there either, ideally it should be Map<String, Object> or something that better represents a JSON object